scripts/test-check-packages.sh

```diff
@@ -69,21 +69,23 @@ for d in test/packages/${PACKAGE_TEST_TYPE:-other}/${PACKAGE_UNDER_TEST:-*}/; do
     elastic-package install -v
 
     if [ "${PACKAGE_TEST_TYPE:-other}" == "benchmarks" ]; then
-      if [ "${PACKAGE_UNDER_TEST:-*}" == "pipeline_benchmark" ]; then
+      # It is not used PACKAGE_UNDER_TEST, so all benchmark packages are run in the same loop
+      package_to_test=$(basename ${d})
+      if [ "${package_to_test}" == "pipeline_benchmark" ]; then
         rm -rf "${OLDPWD}/build/benchmark-results"
         elastic-package benchmark pipeline -v --report-format xUnit --report-output file --fail-on-missing
-        
+
         rm -rf "${OLDPWD}/build/benchmark-results-old"
         mv "${OLDPWD}/build/benchmark-results" "${OLDPWD}/build/benchmark-results-old"
-        
+
         elastic-package benchmark pipeline -v --report-format json --report-output file --fail-on-missing
-        
+
         elastic-package report --fail-on-missing benchmark \
           --new ${OLDPWD}/build/benchmark-results \
           --old ${OLDPWD}/build/benchmark-results-old \
           --threshold 1 --report-output-path="${OLDPWD}/build/benchreport"
       fi
-      if [ "${PACKAGE_UNDER_TEST:-*}" == "system_benchmark" ]; then
+      if [ "${package_to_test}" == "system_benchmark" ]; then
         elastic-package benchmark system --benchmark logs-benchmark -v --defer-cleanup 1s
```

Comment:
@marc-gr now that these are being run, `system_benchmarks` are failing with several errors:

https://buildkite.com/elastic/elastic-package/builds/1040#01890165-f272-4c84-b810-165d22b03e49/95-1974

Could we comment out for the time being this command ?

Reply:
Just talked offline with @marc-gr , it has been decided to comment out this package and not running it for the time being. Applied change here: https://github.com/elastic/elastic-package/pull/1327/commits/f5a280a5f78d0f64804817671958e96fadef40d4